owners/cloud_build.yaml

```diff
@@ -0,0 +1,44 @@
+steps:
+  # The first two steps are required to identify and inject un-redacted secrets.
+  - name: gcr.io/cloud-builders/npm
+    args:
+      - install
+      - yaml
+  - name: gcr.io/cloud-builders/npm
+    args:
+      - run
+      - replace-secrets
+      - --
+      - owners
+    secretEnv:
+      - WEBHOOK_SECRET
+      - PRIVATE_KEY
+      - GITHUB_ACCESS_TOKEN
+
+  # Install app dependencies
+  - name: gcr.io/cloud-builders/npm
+    dir: owners
+    args:
+      - install
+
+  # Deploy the app and its cron tasks
+  - name: gcr.io/cloud-builders/gcloud
+    dir: owners
+    args:
+      - app
+      - deploy
+  - name: gcr.io/cloud-builders/gcloud
+    dir: owners
+    args:
+      - app
+      - deploy
+      - cron.yaml
+
+# These secrets are the base64-encoded form of encrypted secret values. They are
+# automatically decrypted and added to the `.env` environment at build time.
+secrets:
+  - kmsKeyName: projects/ampproject-owners-bot/locations/global/keyRings/amp-github-apps-keyring/cryptoKeys/app-env-key
+    secretEnv:
+      WEBHOOK_SECRET: CiQAs8/dw/BTKMPqcNPIpQZVCdYi6TpBEwAaTuoHbtjfdJWHv28SOQC3YjA4n5HlcR/fSVmVNlB6m1W8lY5FE6oNJFsuqRqtVgirqFQjIN3RVKbiR5NmYKr2OrP0CMA33A==
```

Comment:
Interesting observation - the base64-encoding of all the secrets in all the apps start with `CiQA`. pre-push hook to ban values that don't start with it from being pushed?

Reply:
Indeed an interesting observation, but I'm not sure if that's a bit overboard to have a pre-push hook. Why do you think it's worthwhile here while, say, the environment files would be an easier place to accidentally include a secret?

Reply:
I can see a situation in which one pastes an already base64-encoded value (like the PRIVATE_KEY) without passing it through the encrypter. Extremely minor though...